tempesta_fw/addr.c

```diff
@@ -29,6 +29,69 @@
 #include "lib.h"
 #include "log.h"
 
+static bool
+tfw_addr_eq_inet(const struct sockaddr_in *a, const struct sockaddr_in *b);
+
+static bool
+tfw_addr_eq_inet6(const struct sockaddr_in6 *a, const struct sockaddr_in6 *b);
```

Comment:
Usually we don't use functions forward declarations, rather it's better to place ```tfw_addr_ifmatch()``` after ```tfw_addr_eq_inet()``` and ```tfw_addr_eq_inet6()```